{
  "date": "2025-10-21T12:16:43.748705",
  "num_tests": 1,
  "failures": 0,
  "errors": 1,
  "skipped": 0,
  "successful": false,
  "details": [
    {
      "name": "test_grader (unittest.loader._FailedTest.test_grader)",
      "status": "failed",
      "message": "ImportError: Failed to import test module: test_grader\nTraceback (most recent call last):\n  File \"/opt/anaconda3/lib/python3.12/unittest/loader.py\", line 137, in loadTestsFromName\n    module = __import__(module_name)\n             ^^^^^^^^^^^^^^^^^^^^^^^\nModuleNotFoundError: No module named 'tests.test_grader'\n\n"
    }
  ]
}